src/main.py

```diff
@@ -241,10 +241,17 @@ def main(args):
     parser.add_argument(
         "--amp_level",
         type=str,
-        default="O1",
+        default=None,
         help="The optimization level to use (O1, O2, etc…) for 16-bit GPU precision (using "
         + "NVIDIA apex under the hood).",
     )
+    parser.add_argument(
+        "--amp_backend",
+        type=str,
+        default="apex",
```

Comment:
The default should be `"native"` since PyTorch-Lightning uses that as the default as well.